docs/index.Rmd

```diff
@@ -278,6 +310,19 @@ This partial graph view logically relates **data types** to **assays**.
 
 *Documentation currently in development.*
 
+#### Data Level {#Data .tabset .tabset-fade .tabset-pills}
+
+##### Terms
+
+```{r data_subtypes_table, echo=FALSE}
+
+data_level_table <- schema %>%
```

Comment:
Data Subtype is one of those annotations that I really do not like (in it's current implementation). It's not very explicit. What do you think of adopting more standardized data levels?

Reply:
I don't think dataSubtype is all that useful either. You could probably get rid of it and just use dataType with values that capture the needed information. For example, GDC has data type SimpleSomaticMutation vs. AnnotatedSomaticMutation. You have the information that these are processed data files, but moreover Annotated* is an additional level of "processed", which dataSubtype doesn't even help distinguish. But I think that's a little outside of the scope for now so will figure out how to revamp later.